README.md

```diff
@@ -930,6 +985,28 @@ SPEC_GITHUB_TOKEN=<GitHub password/access token>
 npm test
 ```
 
+**Note**: The standard set of tests doesn't include Bitbucket Server integration testing. To run Bitbucket Server tests, please see the section below.
+
+## Bitbucket Server ##
+
+**Prerequisites**:
+1. A running instance of Bitbucket Server
+1. A clone of [Builder](./) repo placed on this server
```

Comment:
Why is a clone of Builder required to be on the Bitbucket Server?

Reply:
To be precise, the full clone is not needed. Everything needed is a repo with the `spec` directory copied from the Builder repo and placed to the root of the repo. But I think it's simpler and clearer to just clone the full repo.
The `spec` directory is used by tests. So if you want to test Builder against some BB-server, you need to have this dir there.

Reply:
After looking at the test cases, this makes more sense to me - thanks!